py-polars/tests/unit/functions/test_concat.py

```diff
@@ -24,16 +24,19 @@ def test_concat_lf_stack_overflow() -> None:
 
 def test_concat_horizontally_strict() -> None:
     a = pl.DataFrame({"a": [0, 1, 2], "b": [1, 2, 3]})
-    b = pl.DataFrame({"c": [11], "d": [42]}) # 1 vs N
+    b = pl.DataFrame({"c": [11], "d": [42]}) # 1 vs N (may broadcast)
     c = pl.DataFrame({"c": [11, 12], "d": [42, 24]}) # 2 vs N
```

Comment:
Can you rename these to `df1` (1 row), `df2` (2 rows), and `df3` (3 rows) just so it's a little less confusing? :D